pkg/content/cache_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package content
+
+import (
+	"os"
+	"testing"
+
+	"github.com/opencontainers/go-digest"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLeaseCahe(t *testing.T) {
+	lc := newLeaseCache()
+	require.NoError(t, lc.Add("test_1", "1"))
+	require.NoError(t, lc.Add("test_2", "1"))
+	require.NoError(t, lc.Add("test_3", "2"))
+	require.NoError(t, lc.Add("test_2", "2"))
+	require.Error(t, lc.Add("test_3", "test"))
+	require.Equal(t, 3, lc.Len())
+	key, err := lc.Remove()
+	require.NoError(t, err)
+	require.Equal(t, "test_1", key)
+	key, err = lc.Remove()
+	require.NoError(t, err)
+	require.Equal(t, "test_3", key)
+	key, err = lc.Remove()
+	require.NoError(t, err)
+	require.Equal(t, "test_2", key)
+}
+
+func TestLeaseCahePerformance(t *testing.T) {
```

Comment:
How do we test the performance in this test case?

Reply:
Rename to ```TestLeaseCacheInit```.